src/utilities.ts

```diff
@@ -394,85 +394,112 @@ function exeFilename(basename: string): string {
   else return basename;
 }
 
-function findVcastTools() {
-  // This function will check if vPython is on the path or if the
-  // VectorCAST Installation directory option is set
-  // In either of these "positive" cases it will setup the
-  // vPythonCommandToUse and clicastCommandtoUse global variables
-  // otherwise it will display and error.
+function findVcastTools():boolean {
+
+  // This function will set global paths to vpython, clicast and vcastqt
+  // by sequentially looking for vpython in the directory set via the 
+  // 1. extension option: "vcastInstallationLocation"
+  // 2. VECTORCAST_DIR
+  // 3. system PATH variable
+
+
+  // return valuo
+  let foundAllvcastTools = false;
+
+  // value of the extension option
+  const settings = vscode.workspace.getConfiguration("vectorcastTestExplorer");
+  const installationOptionString = settings.get("vcastInstallationLocation", "");
+
+  // value of VECTORCAST_DIR
+  const  vectorcastDirString = process.env ["VECTORCAST_DIR"];
```

Comment:
Maybe rename `vectorcastDirString` to `VECTORCAST_DIR`, it will be clear it's an environment variable later in the code